src/TimeGrid.js

```diff
@@ -189,22 +195,56 @@ export default class TimeGrid extends Component {
             ref={gutterRef}
             className='rbc-time-gutter'
           />
-
-          {this.renderEvents(range, rangeEvents, this.props.now)}
+          {eventsRendered}
 
         </div>
       </div>
     );
   }
+  renderEventsWithResource(range, events, today, resources) {
+    let { min, max, endAccessor, startAccessor, components } = this.props;
+
+    return range.map((date, idx) => {
+      let daysEvents = events.filter(
+          event => dates.inRange(date,
+              get(event, startAccessor),
+              get(event, endAccessor), 'day')
+      )
+
+      return resources.map((resource, id) => {
+
+        let eventsToDisplay = daysEvents.filter(
+            event => event.resourceId === resource.id
```

Comment:
Like the `events` I don't want to be prescriptive about the shape of the data in `resources` so can we add `accessor` props for these fields like we do with startAccessor, et al?